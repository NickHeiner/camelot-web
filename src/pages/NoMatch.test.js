import NoMatch from './NoMatch';
import React from 'React';
import reactTestRenderer from 'react-test-renderer';

describe('NoMatch', () => {
  it('renders', () => {
    expect(reactTestRenderer.create(
            <NoMatch />
        )).toMatchSnapshot();
  });
});
