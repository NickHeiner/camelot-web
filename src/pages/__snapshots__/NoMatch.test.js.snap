exports[`NoMatch renders 1`] = `
<h1>
  404 â€“ Page Not Found
</h1>
`;
